generator/src/routes/+page.ts

```diff
@@ -1,3 +1,43 @@
+import type { Option } from "$components/FormSelectGroup/index.svelte"
+
 // since there's no dynamic data here, we can prerender
 // it so that it gets served as a static asset in production
 export const prerender = true
+
+export const load = () => {
+  const options: Option[] = []
+  for (let i = 0; i < 20; i++) {
+    options.push({
+      id: `${i + 2}`,
+      label: "Item #" + (i + 1),
+    })
+  }
+
+  const productOptions: Option[] = [
+    {
+      id: "arbitrary",
+      label: "Arbitrary",
+    },
+    {
+      id: "premade",
+      label: "Premade",
+    },
+  ]
```

Comment:
This is not a list we want to append things to, this is just a fixed set of toggle buttons - you either enter "arbitrary" values, or you pick a "premade" choice so I would prefer them to be in the code as such.